composer.json

```diff
@@ -24,7 +24,11 @@
     "embed/embed": "^2.6"
   },
   "require-dev": {
-    "phpunit/PHPUnit": "~4.8"
+    "phpunit/PHPUnit": "~4.8",
+    "silverstripe/behat-extension": "dev-master",
+    "silverstripe/serve": "dev-improve-portability",
```

Comment:
do we want to clean up this version constraint before merge? As well as not using your fork and/or registering it with packagist?